hypertrace-trace-enricher/hypertrace-trace-enricher-api/src/main/java/org/hypertrace/traceenricher/trace/util/GraphBuilderUtil.java

```diff
@@ -0,0 +1,40 @@
+package org.hypertrace.traceenricher.trace.util;
+
+import org.hypertrace.core.datamodel.StructuredTrace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GraphBuilderUtil {
```

Comment:
Is it possible to add unit tests for this util?

Reply:
Done.